pkg/storage/driver/driver.go

```diff
@@ -39,44 +39,44 @@ type Creator interface {
 	Create(key string, rls *rspb.Release) error
 }
 
-// Updator is the interface that wraps the Update method.
+// Updater is the interface that wraps the Update method.
```

Comment:
I'm happy to drop such things, however, ... https://wikidiff.com/updater/updator